packages/stencil-library/src/components/icon/icon.tsx

```diff
@@ -0,0 +1,44 @@
+import { Component, getAssetPath, h, Prop } from "@stencil/core";
+
+@Component({
+    tag: 'njwds-icon'
+})
+export class Icon {
+    @Prop() icon: string
+    @Prop() size: "3" | "4"| "scale" = "3"
+    @Prop() decorative: boolean = false;
+    @Prop() iconTitle?: string
+
+    componentWillLoad() {
```

Comment:
[Dust] Any reason why this falls in `componentWillLoad` vs. another lifecycle method?